mlflow/ml-package-versions.yml

```diff
@@ -491,15 +491,15 @@ paddle:
   package_info:
     pip_release: "paddlepaddle"
   models:
-    minimum: "2.4.1"
```

Comment:
Confirmed 2.4.1 has been removed due to the project size limit.

https://github.com/PaddlePaddle/Paddle/issues/71938

Reply:
The mlflow project size is 4.3 GB. We'll be fine for a while.

<img width="480" alt="Screenshot 2025-03-27 at 12 04 31" src="https://github.com/user-attachments/assets/43dbe1a7-9451-4f00-8fde-02fd750b228c" />